lib/nearley.js

```diff
@@ -338,6 +337,73 @@
         return this;
     };
 
+
+    Parser.prototype.reportError = function(token) {
+        var lines = [];
+        const tokenDisplay = (token.type ? token.type + " token: " : "") + JSON.stringify(token.value !== undefined ? token.value : token);
+        lines.push(this.lexer.formatError(token, "Syntax error"));
+        lines.push(`Unexpected ${tokenDisplay}. Instead, I was expecting to see one of the following:\n`);
```

Comment:
Can these be de-sugared out of the backtick-formatted-string syntax? We've tried to keep nearley's core in "old-style" JS throughout and this would be a weird deviation.

Reply:
Done.